indicators/banco-de-la-republica-g5d6u78z.yml

```diff
@@ -11,13 +11,16 @@ detection:
   form:
     html|contains:
       - action="index2.php"
+      
+  image:
+    requests|contains: 'selectArrowDown.b3a49a7d.svg'
 
   css:
     html|contains|all:
       - 2.d18bb301.chunk.css
       - main.8d29879f.chunk.css
 
-  condition: form and css
+  condition: (form and css and image) or (form and css)
```

Comment:
This is equivalent to just `(form and css)`, is the extra `image` condition useful?

Reply:
> This is equivalent to just `(form and css)`, is the extra `image` condition useful?

Sorry for the late reply, I was not home the past month. 
The image is a useful indicator for this kit.